partials/header.hbs

```diff
@@ -28,21 +27,21 @@
         </div>
     {{/if}}
     <div class="header-right hidden-xs hidden-sm hidden-md">
-        {{#if @custom.content_api_key_for_search}}
-            <button class="button-icon menu-item menu-item-button js-modal" data-modal="search" aria-label="Search">
-                <svg class="icon">
-                    <use xlink:href="#magnify"></use>
-                </svg>
-            </button>
-        {{/if}}
-        {{navigation}}
+        <button class="button-icon js-modal" data-modal="search" aria-label="Search">
+            <svg class="icon">
+                <use xlink:href="#magnify"></use>
+            </svg>
+        </button>
+        <a class="menu-item menu-item-impact">
+            {{navigation}}
+        </a>
```

Comment:
I think theres no need to nest the `{{navigation}}` in an `a` element?
Since `{{navigation}}` templates in the other links in HTML already
May make it more confusing to read in the long run

![image](https://user-images.githubusercontent.com/18213722/204695335-62a62a97-f139-4a22-ab4a-c418c5673e2f.png)